// [[Reason: undefined]] ////

//// [tests/cases/conformance/es6/computedProperties/computedPropertyNamesOnOverloads_ES5.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -5,6 +5,5 @@
 declare var accessorName: string;
 declare class C {
     [methodName](v: string): invalid;
     [methodName](): invalid;
-    [methodName](v?: string): invalid;
 }
